jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/TimestampNotificationDead.java

```diff
@@ -0,0 +1,38 @@
+package org.dcsa.jit.persistence.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import lombok.*;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter(AccessLevel.PRIVATE)
+@Entity
+@Table(name = "timestamp_notification_dead")
+@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
```

Comment:
is there a benefit of using the jsonb type opposed to the PostgreSQL TEXT. Are we using the json message for queries?

Reply:
Changed this to TEXT.